test/e2e/create_test.go

```diff
@@ -146,7 +146,15 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 	if gc.numberOfBYOHNodes == 0 {
 		t.Skip("BYOH testing disabled")
 	}
-	_, err := tc.createWindowsMachineSet(gc.numberOfBYOHNodes, false)
+
+	// Scale the Cluster Machine Approver Deployment pods to 0
+	// This is required for testing BYOH CSR approval feature so that BYOH instances
+	// CSR's are not approved by Cluster Machine Approver
+	expectedPodCount := int32(0)
+	err := tc.scaleMachineApproverDeployment(&expectedPodCount)
```

Comment:
How is this working? Won't CVO stomp on this change? I also don't understand why @sebsoto did not have to do this in https://github.com/openshift/windows-machine-config-operator/pull/446

Reply:
Thanks for the explanation. We should add the override in this spot.